src/api/guardian.js

```diff
@@ -1,15 +1,30 @@
 const Guardian = require("guardian-js");
 const guardian_filter = require("./guardian_filter");
+
 const guardianKey = (query, cb) => {
   const welvonAPI = new Guardian(process.env.GUARDIAN_KEY, false);
-
+  
   welvonAPI.content
     .search(query, {
       "show-fields": "headline,trailText,thumbnail,bodyText,page,lastModified"
     })
     .then(response => {
-      cb(JSON.parse(response.body.response));
+      cb(JSON.parse(response.body));
     });
 };
 
-module.exports = guardianKey;
+const guardianItem = (id, cb) => {
+  const api = new Guardian(process.env.GUARDIAN_KEY, false);
+  api.item
+    .search(id, {
+      "show-fields": "bodyText"
+    })
+    .then(response => {
```

Comment:
This function could be written on one line

Reply:
I've changed it a little bit and not sure how to reduce it more... please check update.

Reply:
Is this update still to be pushed?